packages/web-app-media-viewer/src/App.vue

```diff
@@ -27,12 +27,9 @@
         </div>
       </transition>
     </div>
-    <oc-spinner
-      v-if="loading"
-      :aria-label="$gettext('Loading media')"
-      class="uk-position-center"
-      size="xlarge"
-    />
+    <div class="uk-position-center">
```

Comment:
```suggestion
    <div v-if="loading" class="uk-position-center">
```

Reply:
ah yep, good point =)

Reply:
Hehe, now you removed it from the spinner but didn't add it to the div 😁

Reply:
I accepted the propsed code suggestion (from github), sorry let me fix it properly then.

Reply:
Afaik you can only make suggestions on single lines. So I had two different suggestions, one for adding it on the div (which is not committed yet) and one for removing it from spinner (which is committed). Sorry for the confusion

Reply:
Hmm not sure about that - I have updated the original commit let me know if that's more to your liking.